src/main/java/com/redhat/labs/lodestar/participants/resource/ParticipantResource.java

```diff
@@ -71,10 +71,13 @@ public Response getParticipants(@QueryParam("engagementUuids") List<String> enga
     @APIResponses(value = { @APIResponse(responseCode = "202", description = "The request was accepted and will be processed.") })
     @Operation(summary = "Refreshes database with data in git, purging first")
     public Response refreshData() {
+
         participantService.purge();
         participantService.refresh();
         
-        return Response.accepted().build();
+        long participantCount = participantService.getParticipantCount();
+        
+        return Response.ok().header("x-total-participants", participantCount).build();
```

Comment:
looks like this change from accepted to ok might be causing a test failure.

Reply:
yep. a little sloppy. Should be fixed now.